guides/source/caching_with_rails.md

```diff
@@ -3,7 +3,8 @@
 Caching with Rails: An Overview
 ===============================
 
-This guide is an introduction to speeding up your Rails application with caching.
+This guide is an introduction to speeding up your Rails application with caching,
+by stopping your application needing to do the same thing multiple times.
```

Comment:
This seems like it's more confusing and unnecessary. I may be biased but the next sentence explains caching fairly well.